salt/utils/verify.py

```diff
@@ -432,8 +444,8 @@ def check_max_open_files(opts):
         return
 
     msg = (
-        'The number of accepted minion keys({0}) should be lower than 1/4 '
-        'of the max open files soft setting({1}). '.format(
+        'In file salt-master.service: LimitNOFILE={1} should be 4 times '
+        'the number of accepted minion({0}). '.format(
```

Comment:
I think that it would be better to create a document with suggested ways of doing this, and point to that document, instead of putting such a specific error message in the logs.

I will create an issue for this in a minute.